src/ec/packages/ec-component-message/message.story.js

```diff
@@ -15,77 +18,55 @@ import dataWarning from './demo/data--warning';
 import message from './ecl-message.html.twig';
 import notes from './README.md';
 
-const formatIcon = data => {
-  data.icon.path = defaultSprite;
-  data.close.icon.path = defaultSprite;
+const prepareMessage = data => {
+  data.variant = select(
+    'variant',
+    [data.variant],
+    data.variant,
+    buttonLabels.required
+  );
+
+  const name = select(
+    'icon.name',
+    [data.icon.name],
+    data.icon.name,
+    buttonLabels.required
+  );
+  getIconKnobs(data, name, 'notifications', 'l', 'primary', 'none');
+
+  data.title = text('title', data.title, buttonLabels.required);
+
+  data.description = text(
+    'description',
+    data.description,
+    buttonLabels.required
+  );
+
+  data.close.icon.path = select(
```

Comment:
I think we need also something else for the close button, or maybe we should set it in the template, i'm referring to the icon.size for the close button...

Reply:
Sure, updated, I think it's best to have it in the template as you suggested.

Reply:
ook